amr-wind/wind_energy/actuator/wing/ActuatorWing.H

```diff
@@ -37,6 +37,24 @@ struct WingBaseData
     //! Pitch angle for the wing
     amrex::Real pitch{0.0};
 
+    //! Switch for turning on 2D Gaussian (off in spanwise)
+    bool gauss_2D{false};
+
+    //! Switch for turning off force components of actuator
+    vs::Vector fcoord_flags{1.0, 1.0, 1.0};
```

Comment:
can we make this member var slightly more verbose? `force_coord_flags` ? And it might make more sense to make it a `Vector<int>`?